core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-publish-variant/dot-experiments-publish-variant.component.ts

```diff
@@ -0,0 +1,77 @@
+import { CommonModule } from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    EventEmitter,
+    OnInit,
+    Output,
+    ViewChild
+} from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+import { ButtonModule } from 'primeng/button';
+import { ConfirmDialogModule } from 'primeng/confirmdialog';
+import { TableModule } from 'primeng/table';
+import { TagModule } from 'primeng/tag';
+
+import { DotDialogActions, DotDialogComponent } from '@components/dot-dialog/dot-dialog.component';
+import { DotDialogModule } from '@components/dot-dialog/dot-dialog.module';
+import { DotMessageService } from '@dotcms/data-access';
+import { DEFAULT_VARIANT_ID, DotResultSimpleVariant } from '@dotcms/dotcms-models';
+import { DotMessagePipeModule } from '@pipes/dot-message/dot-message-pipe.module';
+
+@Component({
+    selector: 'dot-experiments-publish-variant',
+    templateUrl: './dot-experiments-publish-variant.component.html',
+    styleUrls: ['./dot-experiments-publish-variant.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    imports: [
+        CommonModule,
```

Comment:
You can import what you need from CommonModule separately instead of importing the whole module. 

Like NgIf, NgSwitch, NgFor...

Reply:
done